topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,743 @@
+---
+layout: tutorial_hands_on
+
+title: Identification of allelic variants in SARS-CoV-2 from deep sequencing reads
+level: Introductory
+zenodo_link: "https://zenodo.org/record/5036687"
+questions:
+- How can we extract annotated allelic variants in SARS-Cov-2 sequences in Galaxy?
+- Which tools can we use to identify SARS-CoV-2 clades in Galaxy?
+objectives:
+- Repeat SARS-CoV-2 data preparation
+- Select and execute workflow to extract annotated allelic variants from FASTQ files
+- Execute workflow to summarize annotated allelic variants
+- Interpret summaries for annotated allelic variants
+- Execute workflow to extract consensus sequences
+- Execute tools to assign clades/lineages
+time_estimation: 3H
+key_points:
+- 4 workflows are possible to extract annotated allelic variants from FASTQ files depending on the type of input
+- Many datasets can be processed in parallel using collections
+- Annotated allelic variants can be used to identify SARS-CoV-2 clades/lineages in each samples using consensus sequences
+contributors:
+- wm75
+- bebatut
+tags:
+- covid19
+---
+
+
+# Introduction
+{:.no_toc}
+
+Effectively monitoring global infectious disease crises, such as the COVID-19 pandemic, requires capacity to generate and analyze large volumes of sequencing data in near real time. These data have proven essential for monitoring the emergence and spread of new variants, and for understanding the evolutionary dynamics of the virus.
+
+Two sequencing platforms in combination with several established library preparation strategies are predominantly used to generate SARS-CoV-2 sequence data. However, data alone do not equal knowledge: they need to be analyzed. The Galaxy community developed analysis workflows to support the **identification of allelic variants (AVs) in SARS-CoV-2 from deep sequencing reads**. 
+
+These workflows allow one to identify AVs and lineages in SARS-CoV-2 genomes with variant allele frequencies ranging from 5% to 100% (i.e., they detect variants with intermediate frequencies as well.
```

Comment:
What are AV? long version here?

Reply:
Can we add a link to covid19.galaxyproject.org here ... more information, more benchmarks etc..